references/depth/stereo/presets.py

```diff
@@ -12,20 +12,36 @@ def __init__(
         resize_size: Optional[Tuple[int, int]] = None,
         max_disparity: Optional[float] = 512,
```

Comment:
I think it would be best to make a full valid mask when receiving a `None`. Reason for this is that when merging a dataset that has masks and one that doesn't have masks then the dataloader will fail. My proposal for the `None` case would be to return a `torch.ones_like(img[0])`. 

Also this would have consistent mathematical behaviour with an inexistent masks since we'll be evaluating the loss on all pixels.